components/ui-shell/index.ts

```diff
@@ -15,7 +15,8 @@ import './header/index.js';
 import { qiskitLogoIcon } from '../icons/qiskit-logo.js';
 import { userIcon } from '../icons/user.js';
 import styles from './index.scss';
-import { NavItem, TopLevelNavItem, Variant, NAV_ITEMS } from './settings.js';
+import { Variant, NAV_ITEMS, SOCIAL_LINKS } from './settings.js';
+import type { NavItem, TopLevelNavItem } from './settings.js';
```

Comment:
Saw errors when trying to import type from the same statement 👇 

```ts
import { 
  Variant, 
  NAV_ITEMS, 
  SOCIAL_LINKS, 
  type NavItem,
  type TopLevelNavItem 
} from './settings.js';
```

Reply:
That's right... 🤔

<img width="637" alt="Screen Shot 2022-09-14 at 11 12 45 AM" src="https://user-images.githubusercontent.com/1007051/190196134-2dd04d35-cf1e-43d2-b960-0edd1a6fbe4b.png">